docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # pre-build stage
-FROM ruby:3.2.2-alpine3.16 AS pre-builder
+FROM ruby:3.2.2-alpine3.18 AS pre-builder
```

Comment:
Also why is Ruby's docker image being updated, too?

Reply:
Same here, I was just checking which PR fixed the issues, and figured out it was this, so I just made cherrypick of the PR referenced on description
https://github.com/chatwoot/chatwoot/pull/7759